input/fsh/research-consent-profile.fsh

```diff
@@ -63,14 +52,9 @@ Description: "The obligation of the consent."
 * extension contains 
     type 1..1 and
     reference 0..* and
-    period 0..1 and
```

Comment:
This was added to represent "Time constraints" in table X
![image](https://github.com/mojitoj/ASU-research-consent-on-FHIR/assets/5555612/2b487197-a788-4555-9842-44ed3f5b2b0a)

Reply:
I understood that. But the obligation is already inside a 'provision' and the provision already has a 'period' attribute so I think this is a bit redundant.

Reply:
I moved obligation extension out of provision in a PR, since when I created an example with fsh code, I found  I didn't know how to assign obligations to each provision for this example: https://www.dropbox.com/home/HL7%20FHIR%20Research%20Consent%20Project/Paper?di=left_nav_browse&preview=Figure+-+Consent+Form+Mapping.pptx

Reply:
can't we just use a different provision for each obligation with its own period attribute?